mkosi/backend.py

```diff
@@ -828,9 +832,9 @@ def install_grub(args: MkosiArgs, root: Path, loopdev: Path, grub: str) -> None:
     run_workspace_command(args, root, cmdline, nspawn_params=nspawn_params)
 
 
-def die(message: str) -> NoReturn:
+def die(message: str, exception: Type[MkosiException] = MkosiException) -> NoReturn:
```

Comment:
According to [PEP 484](https://www.python.org/dev/peps/pep-0484/#type-definition-syntax) "Expressions whose type is a subtype of a specific argument type are also accepted for that argument."

But, [this may break compatibility with python 2](https://stackoverflow.com/questions/4015417/why-do-python-classes-inherit-object/45062077#45062077).

Reply:
I'm kinda surprised that works but OK